seckill-event-store/src/main/java/io/servicecomb/poc/demo/seckill/repositories/SpringBasedCouponEventRepository.java

```diff
@@ -21,5 +21,8 @@
 import org.springframework.data.repository.PagingAndSortingRepository;
 
 public interface SpringBasedCouponEventRepository extends PagingAndSortingRepository<PromotionEvent<String>, Integer>, CouponEventRepository {
+  PromotionEvent<String> findTopByCouponIdAndTypeOrderByIdDesc(String couponId, String type);
+  List<PromotionEvent> findByCustomerId(String customerId);
```

Comment:
event store shall be private to command service and events are sent to query service as messages